swift/Sources/Addresses/CoinType+Address.swift

```diff
@@ -41,7 +40,10 @@ public extension CoinType {
         case .ontology:
             return OntologyAddress(string: string)
         case .xrp:
-            return RippleAddress(string: string)
+            if let address = RippleAddress(string: string) {
+                return address
```

Comment:
is there a way to combine into one class? Just `RippleAddress`

Reply:
App won't notice this change, this is same pattern as we did in BitcoinAddress and SegwitAddress, quite different

Reply:
I was just curious in terms of simplicity to handle both in one class, as it's just one address

Reply:
that would work too, have to introduce an new address type